src/backend/src/prisma/schema.prisma

```diff
@@ -419,6 +435,7 @@ model Receipt {
 
 model Reimbursement_Request {
   reimbursementRequestId String                  @id @default(uuid())
+  identifier             Int                     @unique @default(autoincrement())
```

Comment:
weird that this is showing up, merging in develop should fix it

Reply:
I don't remember doing this so...

Reply:
yeah I know, it probably just got in here when you were moving around branches, if you merge in develop, github will recognize that this is already in here